File: src/main/java/ash/java/graphql/types/tvseason/TvCrewType.java
package ash.java.graphql.types.tvseason;

import graphql.annotations.GraphQLField;
import graphql.annotations.GraphQLName;

@GraphQLName("TvCrew")
public class TvCrewType extends TvPersonType {

    @GraphQLField
    private String department;

    @GraphQLField
    private String job;

    public String getDepartment() {
        return department;
    }

    public void setDepartment(String department) {
        this.department = department;
    }

    public String getJob() {
        return job;
    }

    public void setJob(String job) {
        this.job = job;
    }
}

File: src/main/kotlin/ash/kotlin/graphql/types/tvseason/TvCrewType.kt
package ash.kotlin.graphql.types.tvseason

import graphql.annotations.*

@GraphQLName("TvCrew")
class TvCrewType(id: Int = 0) : TvPersonType(id)
{

  @GraphQLField
  var department = ""

  @GraphQLField
  var job = ""
}
